tests/bv32.rs

```diff
@@ -3,3 +3,88 @@
 !*/
 
 ruler::impl_bv!(32);
+
+#[cfg(test)]
+mod test {
+    use super::*;
+    use ruler::enumo::{Filter, Metric, Ruleset, Workload};
+    use std::fs::OpenOptions;
+    use std::io::Write;
+    use std::time::Instant;
+
+    // #[test]
+    fn bv32_oopsla_equiv() {
+        let mut all_rules = Ruleset::default();
+        let start = Instant::now();
+
+        let initial_vals = Workload::new(["a", "b", "c"]);
+        let layer_1 = Workload::make_layer(
+            initial_vals.clone(),
+            &["~", "-"],
+            &["&", "|", "*", "--", "+"],
```

Comment:
in all of these recipes, can you declare the unary ops and binary ops just once so if we add/remove ops it would only need to be updated in one place?